tests/test_classifier.py

```diff
@@ -115,3 +121,34 @@ def test_reasonable_predictions(self):
         model.fit(trX, trY)
         predY = model.predict(teX)
         self.assertEqual(accuracy_score(teY, predY), 1.00)
+
+    def test_language_model(self):
+        """
+        Ensure saving + loading does not cause errors
+        Ensure saving + loading does not change predictions
+        """
+        save_file_autosave = 'tests/saved-models/autosave_path'
+        model = LanguageModelClassifier(verbose=False, autosave_path=save_file_autosave)
+        lm_out = model.lm_predict(5)
+        self.assertEqual(type(lm_out), str)
+        lm_out_2 = model.lm_predict(seed_text="Indico RULE")
+        self.assertEqual(type(lm_out_2), str)
+        self.assertIn('_start_Indico RULE'.lower(), lm_out_2)
```

Comment:
Maybe we could trim the special tokens (i.e. __start__) as part of the lm output?

Reply:
I'm not sure about this. Because its nice to know where the delimiter is in the middle. So trimming some and not all just feels inconsistent.

Reply:
Seems like a reasonable enough justification to me.